common/src/cams/test-utilities/mock-user.ts

```diff
@@ -1,52 +1,80 @@
-import { CamsRole, CamsUser } from '../session';
-import { BUFFALO, MANHATTAN } from './mock-data';
+import { CamsRole } from '../roles';
+import { CamsUser } from '../users';
+import { BUFFALO, MANHATTAN } from './offices.mock';
 
 export type MockUser = {
   sub: string;
   label: string;
   user: CamsUser;
 };
+export const SUPERUSER = {
```

Comment:
Consider how these mock users may relate to mock attorneys in the `attorneys.mock.ts` file.